### PR TITLE
Localization added

### PR DESCRIPTION
**Pull request by aminbenarieb**
Whats' new:
- Russian localization support.
- Markdown for all messages
- Fixes order of sending answer’s result and next question
- Added greeting on start (https://github.com/it-muslim/arabic-verbs-telegram-bot/issues/5).
- Global namespace for localization and other utils.

@rguliev  @Jihadik Check out please the new added namespace. Some functionality should be moved to a separate place, so I've moved it, but not sure is it a best way for this.

~As a next improvement I see replace current localization with https://www.npmjs.com/package/i18n.~ (Done).

**Test Plan**:
```
Amine bin Arieb, [30 Jan 2019 at 15:29:08]:
/start

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:08]:
Ассаляму алейкум, Amine! Данный бот разработан участниками сообщества IT Muslim и предназначен для тренировки спряжений арабских глаголов. Пользователю предлагается выбрать правильную форму глагола в зависимости от рода, времени и числа. Напишите /play для начала тренировки!

Amine bin Arieb, [30 Jan 2019 at 15:29:15]:
/language en

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:16]:
Successfully changed the language to "en" ✅

Amine bin Arieb, [30 Jan 2019 at 15:29:17]:
/start

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:17]:
As-salamu alaykum, Amine! This bot was developed by members of the IT Muslim community and is designed to train the conjugations of the Arabic verbs. Users are invited to choose the correct form of the verb depending on the gender, time and number. Write /play to start a training!

Amine bin Arieb, [30 Jan 2019 at 15:29:19]:
/play

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:20]:
You (two men) did

Amine bin Arieb, [30 Jan 2019 at 15:29:21]:
فَعَلتِ

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:21]:
Oops, the right answer is: فَعَلتُما

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:22]:
You (many men) did

Amine bin Arieb, [30 Jan 2019 at 15:29:25]:
فَعَلتُم

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:26]:
Well done!

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:26]:
She did

Amine bin Arieb, [30 Jan 2019 at 15:29:32]:
/language ru

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:32]:
Язык успешно изменен на "ru" ✅

Amine bin Arieb, [30 Jan 2019 at 15:29:33]:
فَعَلتُما

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:33]:
Упс, правильный ответ: فَعَلَت

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:33]:
Я сделал

Amine bin Arieb, [30 Jan 2019 at 15:29:35]:
فَعَلَت

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:35]:
Упс, правильный ответ: فَعَلتُ

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:36]:
Вы (женщины) сделали

Amine bin Arieb, [30 Jan 2019 at 15:29:39]:
/language ar

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:39]:
Извините, мы пока не поддерживаем "ar" 😞

Amine bin Arieb, [30 Jan 2019 at 15:29:41]:
فَعَلتُنَّ

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:41]:
Отлично!

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:41]:
Вы (мужчины) сделали

Amine bin Arieb, [30 Jan 2019 at 15:29:44]:
فَعَلوا

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:44]:
Упс, правильный ответ: فَعَلتُم

Amin Benarieb Test Bot, [30 Jan 2019 at 15:29:44]:
Вы (двое мужчин) сделали
```

### CODE REVIEW

File: itMuslim.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+let ITMuslim = {}
+
+ITMuslim.localizeString = function (string) {
+  if (typeof (string) !== 'string') {
+    return null
+  }
+
+  const size = Object.keys(arguments).length
```
[the remark this answers is not in the document]
Comment (aminbenarieb) [reply itMuslim.js]: Nope, arguments could me any length.

File: bot.js [hunk 2]
```diff
@@ -12,45 +14,73 @@ if (process.env.NODE_ENV === 'production') {
...
+  let str = msg.from.language_code.toLowerCase()
+  let lgCode = str.substring(0, 2) // dropping region code
+  lgCode = ITMuslim.Language.hasOwnProperty(lgCode) ? lgCode : storage.base_language
+  storage[chatID] = { language_code: lgCode }
+}
+
 function postNewQuestion (chatID) {
-  const question = quiz.generateQuestion('en')
-  storage[chatID] = { 'answer': question.answer }
+  const lgCode = (storage[chatID] && storage[chatID].language_code) || storage.base_language
```
[the remark this answers is not in the document]
Comment (aminbenarieb) [reply bot.js]: Maybe, but i decided to remove prepareUser for now, maybe it future we could return it.

File: bot.js [hunk 3]
```diff
@@ -1,57 +1,138 @@
 const token = process.env.TOKEN
 const Bot = require('node-telegram-bot-api')
 const quiz = require('./quiz.js')
+const i18n = require('i18n')
+const defaultLocale = 'en'
+const supportedLocales = ['en', 'ru']
 
-let bot
+let bot = createBot()
```
Comment (Jihadik) [inline bot.js]: Let's move this down after the definition of `createBot` function?

File: bot.js [hunk 4]
```diff
@@ -1,57 +1,138 @@
...
   bot.setWebHook(process.env.HEROKU_URL + bot.token)
-} else {
-  bot = new Bot(token, { polling: true })
+  return bot
 }
 
-function postNewQuestion (chatID) {
-  const question = quiz.generateQuestion('en')
-  storage[chatID] = { 'answer': question.answer }
+function localeForMsg (msg) {
```
Comment (Jihadik) [inline bot.js]:
```suggestion
function localeFromMsg (msg) {
```

File: bot.js [hunk 5]
```diff
@@ -1,57 +1,138 @@
...
+  const chatID = msg.chat.id
+  let locale = msg.from.language_code.toLowerCase()
+  if (supportedLocales.includes(locale) === false) {
```
Comment (Jihadik) [inline bot.js]:
```suggestion
  if (!supportedLocales.includes(locale)) {
```

File: bot.js [hunk 6]
```diff
@@ -1,57 +1,138 @@
...
+    locale = defaultLocale
+  }
+  return (storage[chatID] && storage[chatID].locale) || locale
+}
+
+function postNewQuestion (msg) {
```
Comment (Jihadik) [inline bot.js]: Let's avoid giving unnecessary info
```suggestion
function postNewQuestion (chatID, locale) {
```

File: bot.js [hunk 7]
```diff
@@ -1,57 +1,138 @@
...
     { parse_mode: 'Markdown' }
   )
 })
 
 bot.onText(/^\/play$/, (msg) => {
-  postNewQuestion(msg.chat.id)
+  postNewQuestion(msg)
+})
+
+bot.onText(/\/setLanguage (.+)/, (msg, match) => {
```
Comment (Jihadik) [inline bot.js]: Users are not programmers :)
```suggestion
bot.onText(/\/language (.+)/, (msg, match) => {
```

File: bot.js [hunk 8]
```diff
@@ -1,57 +1,138 @@
...
+  const chatID = msg.chat.id
+  const userlocale = match[1]
```
Comment (Jihadik) [inline bot.js]:
```suggestion
  const userLocale = match[1]
```

**Pull request merged**